src/Wpf.Ui/Controls/IconSources/SymbolIconSource.cs

```diff
@@ -0,0 +1,129 @@
+﻿// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file, You can obtain one at https://opensource.org/licenses/MIT.
+// Copyright (C) Leszek Pomianowski and WPF UI Contributors.
+// All Rights Reserved.
+
+using System.Windows;
+using System.Windows.Controls;
+using Wpf.Ui.Common;
+using Wpf.Ui.Controls.IconElements;
+
+namespace Wpf.Ui.Controls.IconSources;
+
+/// <summary>
+/// Represents an icon source that uses a glyph from the specified font.
+/// </summary>
+public sealed class SymbolIconSource : IconSource
```

Comment:
Someone, someday, somehow, might want to alter this class. I believe that there should be no sealed classes in an open-source project.  We should open all such classes.